.markdownlint-cli2.jsonc

```
@@ -0,0 +1,164 @@
+{
+  "config": {
+    "default": true,
+    // Some Callouts currently include headings that don't follow this rule
```

As the rule names are not self-explanatory, I would suggest repeating their description here:

```suggestion
    // MD001 - Heading levels should only increment by one level at a time.
    // -> Disabled, as some callouts include headings.
```